stable/ceph-exporter/templates/_helpers.tpl

```diff
@@ -0,0 +1,17 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Fix formatting issues around braces and truncate to 63 chars. Ideally use `helm create` with Helm 2.5 to get a correct `_helpers.tpl`.